test/evaluation/evaluationClientWrapper.spec.ts

```diff
@@ -226,7 +226,7 @@ describe('evaluate', () => {
     pdSchema!.submission_requirements = [pdSchema!.submission_requirements![1]];
     vpSimple!.verifiableCredential![0]!.vc!.issuer = 'did:foo:123';
     const evaluationClientWrapper: EvaluationClientWrapper = new EvaluationClientWrapper();
-    vpSimple!.holder = evaluationClientWrapperData.getHolderDID()[0];
+    vpSimple!.holder = [evaluationClientWrapperData.getHolderDID()[0]];
```

Comment:
seems redundant. `evaluationClientWrapperData.getHolderDID()` returns a single entry list

Reply:
It cannot be a list, it's inside a presentation. Fixed!